styles.css

```diff
@@ -1,44 +1,114 @@
 body {
   font-family: 'ColfaxRegular';
   font-size: 13.5px;
-  color: rgb(0, 51, 85); 
+  color: rgb(0, 51, 85);
 }
 
 h1 {
   font-size: 1.85rem;
   font-family: 'ColfaxWebMedium';
-  padding: 1.7rem;
+}
+
+.difference-section-h1 {
+  padding-top: 3.5rem;
+  padding-bottom: 2.7rem;
+}
+
+.difference-section-h2 {
+  padding: 0.7rem;
 }
 
 h2 {
   font-size: 1rem;
   font-family: 'ColfaxWebMedium';
-  padding: 1rem;
 }
 
 p {
   line-height: 1.5;
 }
 
-section {
-  text-align: center;
-}
-
 img {
   width: 90%;
 }
 
 a {
-  color: rgba(82, 158, 226, 1);
+  color: #4da6e5;
 }
 
-.grid {
+#difference-section {
   padding-left: 9.5rem;
   padding-right: 9.5rem;
+  text-align: center;
 }
 
 .difference-footer-div {
-  padding: 2.5rem;
+  padding-top: 2.5rem;
+  display: inline-block;
+  padding-bottom: 6rem;
+}
+
+#confidence-section-div-left {
+  margin-left: 8.5rem;
+  margin-top: 10rem;
+  text-align: left;
+  color: white;
+}
+
+#confidence-section-div-left p {
+  line-height: 1.5;
```

Comment:
I believe your `p` tags should already have this line height defined above. Also, the font size should be inherited from the body.